src/index.html

```diff
@@ -17,6 +17,88 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo_img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `nav_list` should be `nav__list` to maintain consistency with BEM naming conventions.